tracpro/orgs_ext/tasks.py

```diff
@@ -140,6 +163,9 @@ def run(self, org_pk):
                 msg = "{}: Time limit exceeded for {}".format(self.__name__, org.name)
                 logger.error(msg)
 
+                failure_count_key = self.get_cache_key(org, FAILURE_COUNT)
+                cache.set(failure_count_key, 1 + cache.get(failure_count_key, 0))
```

Comment:
`cache.incr` isn't atomic on all backends but should be for memcache/redis and would preferable to use here.